### ISSUE TITLE
backend: add new user account for DIdentity

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#40]
## User story
1. As a {Admin of DIdentity}
2. I want / need {to create a user account for new HR-employees}
3. So that {the new HR-employee can do his/her work}

## Acceptance criteria
* backend should process input from frontend (have a look at frontend issue for details on field names)
* new user account stored in database
* email automatically sent (to given email-address via the field input) containing a link to set up an initial password
* 

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
